automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala

```diff
@@ -81,44 +93,16 @@ final class NotebookCustomizationSpec extends FreeSpec
       }
     }
 
-    "should install user specified lab extensions" in {
+    "should give cluster user-specified scopes" in {
       withProject { project => implicit token =>
-        withNewCluster(project, request = defaultClusterRequest.copy(userJupyterExtensionConfig = Some(UserJupyterExtensionConfig(labExtensions = Map("jupyterlab-toc" -> "@jupyterlab/toc"))))) { cluster =>
+        withNewCluster(project, request = defaultClusterRequest.copy(scopes = Set("https://www.googleapis.com/auth/userinfo.email", "https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/source.read_only"))) { cluster =>
           withWebDriver { implicit driver =>
             withNewNotebook(cluster) { notebookPage =>
-              val query = """!jupyter labextension list"""
-              val result = notebookPage.executeCell(query).get
-              result should include("@jupyterlab/toc")
-            }
-          }
-        }
-      }
-    }
+              val query = """! bq query --disable_ssl_validation --format=json "SELECT COUNT(*) AS scullion_count FROM publicdata.samples.shakespeare WHERE word='scullion'" """
 
-    "should allow users to install extensions after cluster creation" in {
```

Comment:
Moved lab-extension-related specs to `LabSpec` and left them commented. We should re-enable when we officially support JupyterLab.